src/main/java/seedu/address/logic/commands/TagCommand.java

```diff
@@ -96,7 +98,15 @@ public CommandResult execute(Model model) throws CommandException {
 
         model.setPerson(personToEdit, editedPerson);
         model.getPersonListControl().refreshPersonListUI();
-        return new CommandResult(String.format(MESSAGE_TAGGED_PERSON_SUCCESS, editedPerson));
+        String output = "";
```

Comment:
Maybe you want to consider abstracting out this part of the code to determine the specific success message into a separate method call getTagSuccessMessage and call it within execute instead? (It'd enhance code quality :))